src/html5/parser.rs

```diff
@@ -112,17 +112,6 @@ macro_rules! current_node {
     }};
 }
 
-macro_rules! current_node_mut {
```

Comment:
ah, this is nice.. there was only one place that still uses this.. and we cannot get rid of ALL the macro's (yet).. but the more we can remove, the better

Reply:
because found an issue in `parser.rs`, which is using the macro, then fixed that issue, the macro is not used, so delete.